CHANGELOG.md

```diff
@@ -2,11 +2,17 @@
 
 All notable changes to this project will be documented in this file.
 
-## [0.12.3]
-_released `2024-10-02`_
+## [0.12.4] - 2024-10-07
+
+### Added
+
+- Deprecation warnings on init of SDK
```

Comment:
Might be worth explaining it a bit more clearly to the user, as a developer point of view it seems very logical but maybe something like this could be clearer for non-maintainers:

Deprecation warnings added, now each version of pasqal-cloud is supported for 1 year after release. If your version is not supported anymore you will get a deprecation warning each time you use the SDK.